PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseResourceConnection.java

```diff
@@ -116,19 +116,17 @@ private static Module getTargetModule(@NotNull RunConfiguration configuration) {
         return null;
     }
 
-    private Map<String, String> initEnv() {
+    private Map<String, String> initEnv(@Nonnull final Project project) {
         final Map<String, String> env = new HashMap<>();
-        final Module module = this.consumer.getModule();
```

Comment:
shall we remove consumer?

Reply:
No. here aim to remove dependency on Intellij Module. it is different with our consumer - ModuleResource.

Reply:
our logic only depend on Project. it's unnecessary to add a "getModule" method on ModuleResource. and it may cause error in multi-projects case.